pkgs/development/python-modules/terminaltables3/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+}:
+
+buildPythonPackage rec {
+  pname = "terminaltables3";
+  version = "4.0.0";
+  pyproject = true;
+
+  propagatedBuildInputs = [
+    poetry-core
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Tj7v4gmqiQBaCjTRUlc5QkVpcp7im15kqN1Rxevat38=";
+  };
+
+  meta = with lib; {
+    description = "Display simple tables in terminals";
+    homepage = "https://github.com/matthewdeanmartin/terminaltables3";
+    license = licenses.mit;
+    maintainers = with maintainers; [ kekschen ];
```

Comment:
Also add yourself to maintainer-list.nix :)

Reply:
Already done. I'm still waiting for #353536